src/_pytest/terminal.py

```diff
@@ -443,8 +443,7 @@ def pytest_runtest_logstart(self, nodeid, location):
             self.write_ensure_prefix(line, "")
             self.flush()
         elif self.showfspath:
-            fsid = nodeid.split("::")[0]
-            self.write_fspath_result(fsid, "")
```

Comment:
This is now different, correct? Or does is it have the same outcome?

Reply:
`write_fspath_result` does this:

```py
fspath = self.config.rootdir.join(nodeid.split("::")[0])
```

(and doesn't use `nodeid` otherwise) so the split is redundant here.